code/datums/gamemode/factions/bloodcult/bloodcult.dm

```diff
@@ -917,6 +917,76 @@ var/static/list/valid_cultpower_slots = list(
 
 	return power
 
+/mob/proc/get_convertibility()
+	if (!mind || isDead())
+		return CONVERTIBLE_NOMIND
+
+	if (iscultist(src))
+		return CONVERTIBLE_ALREADY
+
+	return 0
+
+/mob/living/carbon/get_convertibility()
+	var/convertibility = ..()
+
+	if (!convertibility)
+		//TODO: chaplain stuff
+		//this'll do in the meantime
+		if (mind.assigned_role == "Chaplain")
+			return CONVERTIBLE_NEVER
+
+		var/acceptance = "Never"
+		if (client)
+			acceptance = get_role_desire_str(client.prefs.roles[CULTIST])
+
+		for(var/obj/item/weapon/implant/loyalty/I in src)
```

Comment:
👀👀👀

Reply:
read the damn doc https://docs.google.com/document/d/1e49TxEMFdzAwpvIy6alBgYrytWfAo5F-O5A62PW2KvY/edit#

It won't do it in this PR but it might in the next one, since I want to tackle conversion as early as possible.

Reply:
I've read the doc and liked what I read. This is just too good to be true seeing it in action. Bless you Deity.